### PR TITLE
Week 1 Portfolio 

### PR DESCRIPTION
**Pull request by riku1212**
Update README

### CODE REVIEW

**Comment by riku1212** [comment]
Updated week 1 portfolio

File: portfolio/src/main/webapp/script.js [hunk 1]
```diff
@@ -26,3 +26,28 @@ function addRandomGreeting() {
...
+
+function displayHobbies() {
+    const title = document.getElementById('content-title');
+    title.innerText = "This is a list of my main hobbies:";
+    $(function(){
+        $("#content-panel").load("pages/hobbies.html"); 
+    });
+}
+
+function activateListItem() {
```
Comment (kai-dream) [inline portfolio/src/main/webapp/script.js:46]: Nice. I like the highlighting effect when I click on the item.

File: portfolio/src/main/webapp/script.js [hunk 2]
```diff
@@ -26,3 +26,28 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function displayAbout() {
+    const title = document.getElementById('content-title');
+    title.innerText = "Here is some background information about me:";
+    $(function(){
+        $("#content-panel").load("pages/aboutme.html");
```
Comment (kai-dream) [inline portfolio/src/main/webapp/script.js:34]: cool, you moved this section to a separate file!

**Review by kai-dream — APPROVED** [review]
LGTM +1

File: portfolio/src/main/webapp/script.js (hunk 2 above)
Comment (riku1212) [reply portfolio/src/main/webapp/script.js:34]: thanks!

File: portfolio/src/main/webapp/script.js (hunk 1 above)
Comment (riku1212) [reply portfolio/src/main/webapp/script.js:46]: thanks!

**Pull request merged**